lib/home/view/tasks_page.dart

```diff
@@ -13,16 +13,14 @@ class TasksPage extends StatelessWidget {
       appBar: AppBar(
         title: const Text('To Do List'),
       ),
-      body: Center(
-        child: Container(
-          padding: const EdgeInsets.all(20),
-          height: 200,
-          width: 400,
-          child: TaskCard(
-          /// TODO(kamil): for dev purposes only
-            task: TasksDevData.tasks.first,
-          ),
-        ),
+      body: ListView.builder(
+        itemCount: TasksDevData.tasks.length,
+        itemBuilder: (context, index) {
+          final task = TasksDevData.tasks[index];
```

Comment:
it's pretty good.
one small tip, it's good to declare tasks as final before we start iterate through them to make sure that we iterate on list that will not change. In your case it would not make any harm, but it's just a good standard.

Simply:

final tasks = TasksDevData.tasks;

build(BuildContext context) {
   ...
   final task = tasks[index];
}